backend/dal/Services/Concrete/ProjectService.cs

```diff
@@ -725,31 +727,31 @@ public async System.Threading.Tasks.Task<Project> SetStatusAsync(Project project
                     // This must be done first because it requires its own transaction.
                     var projectNumber = this.Context.GenerateProjectNumber(_options.Project.NumberFormat);
                     this.Context.UpdateProjectNumber(originalProject, projectNumber);
-                    originalProject.SubmittedOn = DateTime.UtcNow;
+                    project.SubmittedOn = DateTime.UtcNow;
```

Comment:
The `originalProject.Merge(project, this.Context);` occurs before this.  Which means that these may not be correctly merged.  Or are all of these `Metadata` and will be handled by the last serialize you've added?